haystack/management/commands/clear_index.py

```diff
@@ -10,6 +10,7 @@
 
 class Command(BaseCommand):
     help = "Clears out the search index completely."
+    stealth_options = ('batchsize', 'workers')
```

Comment:
Hey @acdha , I don't see where these are used at all.  Are they simply left over from a previous life or do I not understand how stuff is used?